package.json

```diff
@@ -93,6 +93,16 @@
                 "when": "editorTextFocus && vim.useCtrlKeys"
             },
             {
+                "key": "ctrl+a",
```

Comment:
Note that there's already a `ctrl+a` action in vscode, which seems to hop between the beginning of the line and the first non-whitespace char on the line.

I don't really want to disable that (least surprising), but people may be confused if they ctrl+a on a line and some text doesn't match so their cursor is moved to the beginning.